api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/Parameter.kt

```diff
@@ -0,0 +1,47 @@
+package org.jetbrains.kotlinx.dl.api.core.layer
+
+import org.jetbrains.kotlinx.dl.api.core.KGraph
+import org.jetbrains.kotlinx.dl.api.core.initializer.Initializer
+import org.jetbrains.kotlinx.dl.api.core.regularizer.Regularizer
+import org.jetbrains.kotlinx.dl.api.core.util.getDType
+import org.tensorflow.Operand
+import org.tensorflow.Shape
+import org.tensorflow.op.Ops
+import org.tensorflow.op.core.Variable
+
+public data class Parameter(
```

Comment:
Please advise on the naming

Reply:
At this moment it's not exactly Parameter.. I suggest to rename to KVariable